### ISSUE TITLE
[Feature] Spellcasting Activations tab

### ISSUE THREAD

**Issue by hunarthebarbarian** [ChasarooniZ/pf2e-item-activations#5]
Could these actions, much like wands and scrolls, be added to the spellcasting activations tab instead of the Actions? the Actions sheet can get kind of bogged down with actions for some players, also it would be cool to cast prestidigitation from my mages hat under the spellcasting activation tab anyway lol, same with lifting belts, ventriloquist rings, etc. Is there a reason you prefer the action tab to the spellcasting activations for this?

**Comment by ChasarooniZ** [comment]
I'd say no, perhaps maybe I can add spellcasting entries for ones that add spells, but for other ones there isn't any way to put actions in the spellcasting tab, and the work required to rewrite that seems like more effort than I'd be able to put in.

Tl;Dr: Will look into adding spellcasting entries, but I do this in my free time so no promises as current method works as is ATM.

**Comment by hunarthebarbarian** [comment]
Cool! I figured it might be rough for non spells after I thought about it for a bit, such as with the Lifting Belts and such. I appreciate the feedback, and more than appreciate you doing this on spare time. I can relate to that kind of passion for this stuff:) Thank you for the quick response as well!

**Comment by ChasarooniZ** [comment]
Focus on Spellhearts as an easy "First thing"

**Comment by ChasarooniZ** [comment]
Added in `13.4.4`
